theme/github-box.css

```
@@ -0,0 +1,143 @@
+html.coal .github-repo-box,
+html.navy .github-repo-box,
+html.ayu .github-repo-box {
+    --gh-bg: #0d1117;
+    --gh-border: #30363d;
+    --gh-text: #c9d1d9;
+    --gh-link: #58a6ff;
+    --gh-muted: #8b949e;
+    --gh-btn-bg: #21262d;
+    --gh-btn-border: rgba(240, 246, 252, 0.1);
+    --gh-btn-hover: #30363d;
+    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.5);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)